catalog/app/components/Dropzone/__snapshots__/FilesStats.spec.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FilesStats should render 1`] = `null`;
```

Comment:
There is already, but it is skipped because `react-intl` breaks `jest`
https://github.com/quiltdata/quilt/blob/c571c1bb004fc5b941c15424d30dbf6d934dbed6/catalog/app/components/Dropzone/FilesStats.spec.js#L12

Reply:
This line causes exception https://github.com/quiltdata/quilt/blob/c571c1bb004fc5b941c15424d30dbf6d934dbed6/catalog/app/utils/string.js#L34

Reply:
probably it's because it requires the Intl context. not a blocker anyways

Reply:
> it requires the Intl context

this